<launch>
  <node name="getid" pkg="cafer_core" type="getid" ns="/cafer_core" output="screen"/>
</launch>
